<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHistory</class>
 <widget class="QDialog" name="CHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>110</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLast">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>110</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPre">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>110</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToDate">
   <property name="geometry">
    <rect>
     <x>397</x>
     <y>110</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2015-08-07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFromDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2015-08-07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFirst">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>110</y>
     <width>180</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPage">
   <property name="geometry">
    <rect>
     <x>1730</x>
     <y>110</y>
     <width>150</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Page(0/0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>경보</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnControl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>제어</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnEvent">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>이벤트</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnSystem">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>시스템</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnInq">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>110</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>조회</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>1920</width>
     <height>991</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageAlarm">
    <widget class="QTableWidget" name="tableAlarm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕_코딩</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>14</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그설명</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>구분</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재상태</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>이전상태</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>발생값</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>사용자</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pageCoontrol">
    <widget class="QTableWidget" name="tableControl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕_코딩</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>14</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>사용자</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그설명</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>제어값</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>결과</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pageEvent">
    <widget class="QTableWidget" name="tableEvent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>14</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>태그설명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>발생값</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pageConfig">
    <widget class="QTableWidget" name="tableConfig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕_코딩</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>14</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>사용자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>항목</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>변경전</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>변경후</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="pageSystem">
    <widget class="QTableWidget" name="tableSystem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>14</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>사용자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>종류</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>내용</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="tableEps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>13</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>내용</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTableWidget" name="tableTms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1900</width>
       <height>980</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>13</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>순번</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>일자</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>내용</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>8</y>
     <width>280</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>설정</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1920</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>8</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>이력</string>
   </property>
  </widget>
  <widget class="NumEdit" name="fromHour">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>110</y>
     <width>60</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>110</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>시 부터</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>110</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>까지</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">numedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>CHistory</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>929</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
